sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.F

```diff
@@ -459,43 +460,35 @@ SUBROUTINE TERSUB(IMN,JMN,IM,JM,NM,NR,NF0,NF1,NW,EFAC,BLAT,
 ! Read 30-sec Antarctica RAMP data. Points scan from South
 ! to North, and from Greenwich to Greenwich.
 
-! The error handling here needs to be cleaned up.
-      iosg = 0
       error=NF__OPEN("./topography.antarctica.ramp.30s.nc",
      &               NF_NOWRITE,fsize,ncid)
       call netcdf_err(error, 'Opening RAMP topo file' )
       error=nf_inq_varid(ncid, 'topo', id_var)
       call netcdf_err(error, 'Inquire varid of RAMP topo')
       error=nf_get_var_real(ncid, id_var, GICE)
-      iosg=error
       call netcdf_err(error, 'Inquire data of RAMP topo')
       error = nf_close(ncid)
       
-      if(iosg .ne. 0 ) then
-        print *,' *** Err on reading GICE record, iosg=',iosg
-        print *,' exec continues but NO GICE correction done '
```

Comment:
is this error not required or is handled elsewhere ?

Reply:
I decided to stop processing if there was an error reading the RAMP data. Previously, if there was an error, processing would continue without this data. That is not a good programming practice. If there is a problem, the user might not notice and get a result he does not expect. Better to stop because this RAMP data is considered important.